package.json

```
@@ -16,10 +16,12 @@
     "clean:install": "pnpm clean:node_modules && pnpm install --frozen-lockfile",
     "build": "pnpm clean:bundle && turbo ready && turbo build",
     "build:firefox": "pnpm clean:bundle && turbo ready && cross-env __FIREFOX__=true turbo build",
-    "zip": "turbo build && pnpm -F zipper zip",
-    "zip:firefox": "turbo build && cross-env __FIREFOX__=true pnpm -F zipper zip",
+    "zip": "pnpm -F zipper zip",
+    "zip:firefox": "cross-env __FIREFOX__=true pnpm -F zipper zip",
     "dev": "turbo ready && cross-env __DEV__=true turbo watch dev --concurrency 20",
     "dev:firefox": "turbo ready && cross-env __DEV__=true __FIREFOX__=true turbo watch dev --concurrency 20",
+    "e2e": "turbo e2e",
+    "e2e:firefox": "cross-env __FIREFOX__=true turbo e2e",
```

Let's add this command chain here, because we follow this approach everywhere now.
If you want, you can refactor it in the future, but for now, back it alive.

```suggestion
    "zip": "pnpm build && pnpm -F zipper zip",
    "zip:firefox": "pnpm build:firefox && cross-env __FIREFOX__=true pnpm -F zipper zip",
    "dev": "turbo ready && cross-env __DEV__=true turbo watch dev --concurrency 20",
    "dev:firefox": "turbo ready && cross-env __DEV__=true __FIREFOX__=true turbo watch dev --concurrency 20",
    "e2e": "pnpm build && turbo e2e",
    "e2e:firefox": "pnpm build:firefox && cross-env __FIREFOX__=true turbo e2e",
```

---

This approach won't work, since `cross-env __FIREFOX__=true` not applied for `pnpm build` part. I thought we discussed that it deserves separate fix probably in turbo.

---

I have been working on solution for that env, give me some time :)